### PR TITLE
rippled account queue stats; Dactyl badges

### PR DESCRIPTION
**Pull request by mDuo13**
Normally I wouldn't batch these two mostly unrelated changes, but I discovered that I needed to link to rippled versions, which were broken recently. (Reference links whose text contains `backticks` don't work in the Python markdown parser.) So that basically presented an opportunity, where I had to change the exact thing anyway, so I changed it to how we wanted it - to use badges.

Also, I did a find/replace to turn tabs into 4 spaces throughout the code samples, and I slightly cleaned up the formatting of the "Response Formatting" section in a non-substantive way.

Basically, sorry this is a slightly messy pull request. The part that really needs reviews is the **account_info** section of the file `content/reference-ledger-format.md` -- everything else should be fine.

### CODE REVIEW

File: content/reference-rippled.md [hunk 1]
```diff
@@ -834,15 +918,37 @@ The response follows the [standard format](#response-formatting), with the resul
 | Field | Type | Description |
 |-------|------|-------------|
 | account_data | Object | The [AccountRoot ledger node](reference-ledger-format.html#accountroot) with this account's information, as stored in the ledger. |
-| signer\_lists | Array | (Omitted unless the request specified `signer_lists` and at least one SignerList is associated with the account.) Array of [SignerList ledger nodes](reference-ledger-format.html#signerlist) associated with this account for [Multi-Signing](reference-transaction-format.html#multi-signing). Since an account can own at most 1 SignerList, this array should always have exactly 1 member if it is present. _(New in [`rippled` 0.31.0][])_ |
+| signer\_lists | Array | (Omitted unless the request specified `signer_lists` and at least one SignerList is associated with the account.) Array of [SignerList ledger nodes](reference-ledger-format.html#signerlist) associated with this account for [Multi-Signing](reference-transaction-format.html#multi-signing). Since an account can own at most 1 SignerList, this array should always have exactly 1 member if it is present. [New in: rippled 0.31.0][] |
```
Comment (ryangyoung) [inline content/reference-rippled.md]: Should `1` be `one`?  MS Style Guide says:

> Spell out zero through nine if the number does not precede a unit of measure or is not used as input.

File: content/reference-rippled.md [hunk 2]
```diff
@@ -834,15 +918,37 @@ The response follows the [standard format](#response-formatting), with the resul
...
 | ledger\_current\_index | Integer | (Omitted if `ledger_index` is provided instead) The sequence number of the most-current ledger, which was used when retrieving this information. The information does not contain any changes from ledgers newer than this one.  |
 | ledger\_index | Integer | (Omitted if `ledger_current_index` is provided instead) The sequence number of the ledger used when retrieving this information. The information does not contain any changes from ledgers newer than this one. |
-| validated | Boolean | True if this data is from a validated ledger version; if omitted or set to false, this data is not final. _(New in [`rippled` 0.26.0][])_ |
+| queue\_data | Object | (Omitted unless `queue` specified as `true`) Information about [queued transactions](concept-transaction-cost.html#queued-transactions) sent by this account. Some information is calculated "lazily" and may not be available. This information describes the state of the local `rippled` server, which may be different from other servers in the consensus network. |
```
Comment (ryangyoung) [inline content/reference-rippled.md]: Does "lazily" mean that the information isn't available or that some additional fetch or computation wasn't performed? The last two sentences are little confusing ("Some information...." and "This information..."). Combining them might help?

File: content/reference-rippled.md [hunk 3]
```diff
@@ -896,8 +1002,8 @@ The request accepts the following paramters:
...
 | ledger\_index | String or Unsigned Integer| (Optional) The sequence number of the ledger to use, or a shortcut string to choose a ledger automatically. (See [Specifying a Ledger](#specifying-ledgers))|
 | peer | String | (Optional) The [Address][] of a second account. If provided, show only lines of trust connecting the two accounts. |
-| limit | Integer | (Optional, default varies) Limit the number of transactions to retrieve. The server is not required to honor this value. Cannot be smaller than 10 or larger than 400. _(New in [`rippled` 0.26.4][])_ |
-| marker | [(Not Specified)](#markers-and-pagination) | (Optional) Server-provided value to specify where to resume retrieving data from. _(New in [`rippled` 0.26.4][])_ |
+| limit | Integer | (Optional, default varies) Limit the number of transactions to retrieve. The server is not required to honor this value. Cannot be smaller than 10 or larger than 400. [New in: rippled 0.26.4][] |
+| marker | [(Not Specified)](#markers-and-pagination) | (Optional) Server-provided value to specify where to resume retrieving data from. [New in: rippled 0.26.4][] |
```
Comment (ryangyoung) [inline content/reference-rippled.md]: Does "server-provided" mean that this can be omitted? Or that you need to get the value from the server before making the request?

File: content/reference-rippled.md [hunk 4]
```diff
@@ -807,23 +811,103 @@ An example of a successful response:
...
+            "index": "92FA6A9FC8EA6018D5D16532D7795C91BFB0831355BDFDA177E86C8BF997985F"
+        },
+        "ledger_current_index": 4,
+        "queue_data": {
+            "auth_change_queued": true,
+            "highest_sequence": 10,
+            "lowest_sequence": 6,
+            "max_spend_drops_total": "500",
+            "transactions": [
+
```
Comment (ximinez) [inline content/reference-rippled.md]: Nit: Blank line here and a couple of lines down.

File: content/reference-rippled.md (hunk 2 above)
Comment (ximinez) [inline content/reference-rippled.md]: > (Omitted unless `queue` specified as `true`)

To be pedantic, "(Omitted unless `queue` specified as `true` and querying the current ledger)"

Comment (ximinez) [reply content/reference-rippled.md]: "Lazily" means that one or more computations haven't been performed. (Internally, we don't bother computing some of the values for queued transactions until a later transaction is submitted. We then do those calculations and cache the results to see if the new tx can also be queued.)

IMHO, the last two sentences would read better if they were swapped.

File: content/reference-rippled.md (hunk 3 above)
Comment (ximinez) [reply content/reference-rippled.md]: @ryangyoung "server-provided" means that it was provided as the output to a previous request that had a `limit`.

**Review by ximinez — APPROVED** [review]
I left a couple of minor suggestions, but this looks good. :+1:

**Comment by mDuo13** [comment]
Test failure is not part of this document. (It should be fixed, but by a different PR.)
